cmd/root.go

```diff
@@ -150,18 +150,27 @@ func setFlags(cmd *cobra.Command, v *viper.Viper) {
 	flags.String(
 		"offline-queue-file",
 		"",
-		"(internal) Specify a offline queue file, which will be used instead of the default one.",
+		"(internal) Specify an offline queue file, which will be used instead of the default one.",
 	)
 	flags.String("plugin", "", "Optional text editor plugin name and version for User-Agent header.")
 	flags.String("project", "", "Override auto-detected project."+
 		" Use --alternate-project to supply a fallback project if one can't be auto-detected.")
+	flags.String(
+		"project-folder",
+		"",
+		"Optional workspace path. Override auto-detected project folder."+
+			" Useful only when --relative-paths is true.")
 	flags.String(
 		"proxy",
 		"",
 		"Optional proxy configuration. Supports HTTPS SOCKS and NTLM proxies."+
 			" For example: 'https://user:pass@host:port' or 'socks5://user:pass@host:port'"+
 			" or 'domain\\user:pass'",
 	)
+	flags.Bool(
+		"relative-paths",
+		false,
+		"When set, left-trim the absolute path removing 'workspace-path'.")
```

Comment:
When set, send the file's path relative to the project folder. For ex: /User/me/projects/bar/src/file.ts is sent as src/file.ts so the server never sees the full path. When the project folder cannot be detected, only the file name is sent. For ex: file.ts.

Reply:
Let's rename this to match the ini setting: `--hide-project-folder`

Reply:
sure